app/javascript/src/HelloWorld/ror_components/HelloWorld.client.jsx

```
@@ -0,0 +1,21 @@
+    <div>
+      <h3>Hello, {name}!</h3>
+      <hr />
+      <form>
+        <label className={style.bright} htmlFor="name">
+          Say hello to:
+          <input id="name" type="text" value={name} onChange={(e) => setName(e.target.value)} />
+        </label>
+      </form>
+    </div>
+  );
+};
```

ESLint

[error] Function component is not a function declaration

(react/function-component-definition)